modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheFullTextQuerySelfTest.java

```diff
@@ -59,6 +63,26 @@
     /** Cache name */
     private static final String PERSON_CACHE = "Person";
 
+    /** Limitation to query response size */
+    private static final int QUERY_LIMIT = 5;
+
+    /** Concurrent threads number */
+    private static final int N_THREADS = 20;
+
+    /**
+     * Container for expected values and all available entries
+     */
+    private static final class TestPair {
+
```

Comment:
Please remove this empty line and add javadocs to class members (automated checks require it).